reV/handlers/sam_resource.py

```diff
@@ -83,6 +115,9 @@ def __init__(self, project_points, time_index, require_wind_dir=False):
             if not require_wind_dir:
                 self._res_arrays['winddirection'] = np.zeros(self._shape,
                                                              dtype='float32')
+        else:
+            raise HandlerValueError(f'Selected tech {project_points.tech} is'
```

Comment:
you should set the error message to a string, do logger.error(e) and also raise the exception.

Reply:
Done